docs/known_issues.md

```diff
@@ -0,0 +1,28 @@
+# Velero Plugin for vSphere Known Issues
+
+This section lists the major known issues with Velero Plugin for vSphere. For a complete list of issues, check the [Github issues](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/issues) page. If you notice an issue not listed in Github issues page, please file an issue on the Github repository.
+
+## vSphere CSI driver config file
+
+On a Vanilla cluster, Velero Plugin for vSphere expects the key `csi-vsphere.conf` being presented in the `data` field of the secret `vsphere-config-secret`. If it is not present, backup will fail.
```

Comment:
Can we mention that this key and secret names are the default when vsphere csi is installed.

Reply:
Done